lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -173,7 +173,7 @@ def test_start_date_render(self):
 
     ## CourseEndDate
 
-    def test_course_end_date_during_course(self):
+    def test_course_end_date_during_course_verified(self):
```

Comment:
Admittedly a minor point, but try to avoid getting hung up on particular modes here. Instead of naming these tests for specific enrollment modes, name them to reflect whether a certificate is at stake.